misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -828,6 +831,49 @@ abstract class PathAssertionsSpec(
         }
     }
 
+    describeFun(isRelative) {
+        val isRelativeFun = isRelative.lambda
+        val expectedMessage = "$isDescr: ${RELATIVE_PATH.getDefault()}"
+
+        context("not accessible") {
+            it("throws an AssertionError for a non-existent path") withAndWithoutSymlink { maybeLink ->
+                val file = maybeLink.create(tempFolder.tmpDir.resolve("nonExistent"))
+                expect {
+                    expect(file).isRelativeFun()
+                }.toThrow<AssertionError>().message {
+                    contains(expectedMessage, FAILURE_DUE_TO_NO_SUCH_FILE.getDefault())
+                    containsExplanationFor(maybeLink)
+                }
+            }
+
+            itPrintsFileAccessProblemDetails { testFile ->
+                expect(testFile).isRelativeFun()
+            }
+        }
+
+        it("throws an AssertionError for absolute path") withAndWithoutSymlink { maybeLink ->
+            val path = maybeLink.create(tempFolder.newFile("test"))
+                .absolutePathAsString
+                .let { File(it) }
+                .toPath()
```

Comment:
I don't know why you want to turn it into an absolutePathAsString and then back to Path but this could be simplified to
```suggestion
            val path = maybeLink.create(tempFolder.newFile("test"))
```
this results in (on linux) `/tmp/spek5529000248902455654/test` in case of no symlink and `/tmp/spek9862516612964987379/__linkTo_test` in case of a symlink. Both are already absolute paths

Reply:
fixed